ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.hbs

```diff
@@ -12,7 +12,6 @@
  **/
  --}}
 <div class="info-feature">{{this.target}}</div>
-<div class="info-coordinates">
-    <span class="coordinate-lat">{{this.lat}}</span>
-    <span class="coordinate-lon">{{this.lon}}</span>
+<div class="info-coordinates"><!--
+    --><span class="coordinate-lat">{{this.lat}}</span> <span class="coordinate-lon">{{this.lon}}</span>
```

Comment:
✏️  Why is there an empty comment in here?

Reply:
I believe it's related to getting things to render in the browser in the way we want them. It's essentially commenting out the newline.